content/general/concepts/boolean/boolean.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: 'Booleans'
+Description: 'The boolean data type.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Booleans hold a true or false value. Booleans are used in conditional statements. Ex:
+
+```javascript
+if (true) {
+  // true would always evaluate to true so the folloing line will always run.
+  console.log('this will always print');
+} else if (false) {
+  // false will alawys evaluate to false so the following lines will never run
+  console.log('This will never print');
+}
```

Comment:
```suggestion
if (42 !== "42") {
  // If the statement above always evaluates to true...
  console.log('This will always print.');
} else {
  console.log('This will never print.');
};
```